client/Otiport-client/src/services/endpoints.js

```diff
@@ -1,9 +1,9 @@
 class Endpoints {
-  baseApiPath = "http://localhost:5000";
```

Comment:
Bunu configurable ve ortama bağlı bir şekilde değişebilir hale getirelim. Bunun için yrı bir efor gerekir. Bunu bir issue olarak açalım